addons/revolutionaries/CfgVehicles_Tanoan.hpp

```diff
@@ -0,0 +1,161 @@
+// Units - Revolutionaries (Tanoan)
+
+class TACU_Revolutionaries_U_O_Tanoan_Base: TACU_Main_U_OPFOR_Soldier_Base {
+    author = "Jack";
+    displayName = "Rifleman (FAL)";
+    faction = "TACU_Revolutionaries_O";
+    scope = 2;
+    scopeCurator = 2;
+    identityTypes[] = {"LanguageFRE_F", "Head_Tanoan", "G_Balaclava_blk"};
+    genericNames = "TanoanMen";
+    icon = "iconMan";
+    role = "Rifleman";
+    uniformClass = "U_C_MAN_casual_1_F";
+    backpack = "TACU_Revolutionaries_Backpack_FNFAL";
+    linkedItems[] = {"ItemWatch","ItemCompass","ItemMap"};
+    respawnLinkedItems[] = {"ItemWatch","ItemCompass","ItemMap"};
+    Items[] = {mag_5("ACE_fieldDressing"),"ACE_EarPlugs"};
+    respawnItems[] = {mag_5("ACE_fieldDressing"),"ACE_EarPlugs"};
+    weapons[] = {"CUP_arifle_FNFAL"};
+    respawnWeapons[] = {"CUP_arifle_FNFAL"};
+    magazines[] = {mag_2("CUP_20Rnd_762x51_FNFAL_M")};
+    respawnMagazines[] = {mag_2("CUP_20Rnd_762x51_FNFAL_M")};
+    editorSubcategory = "TACU_Revolutionaries_EdSubCat_O_Tanoan";
+    // editorPreview = QPATHTOF(ui\Cartel_U_O_Soldado_Rifleman.jpg);
+    headgearList[] = {};
+    TACU_allowedFacewear[] = {
+        "G_Balaclava_blk", 1
+    };
+};
+
+//Unit Specifics
+
+class TACU_Revolutionaries_U_O_Tanoan_Rifleman01: TACU_Revolutionaries_U_O_Tanoan_Base {
+    displayName = "Rifleman (MK14)"; 
+    scope = 2;
+    scopeCurator = 2;
+    uniformClass = "U_C_MAN_casual_2_F";
+    weapons[] = {"srifle_DMR_06_hunter_F"};
+    respawnWeapons[] = {"srifle_DMR_06_hunter_F"};
+    magazines[] = {mag_2("10Rnd_Mk14_762x51_Mag")};
+    respawnMagazines[] = {mag_2("10Rnd_Mk14_762x51_Mag")};
+	backpack = "TACU_Revolutionaries_Backpack_Mk14";
```

Comment:
Same issue with backpack formatting.
```suggestion
    backpack = "TACU_Revolutionaries_Backpack_Mk14";
```